.travis.yml

```diff
@@ -20,13 +20,6 @@ env:
     - HOST_CLOUD=openstack
     - HOST_CLOUD=azure
 
-addons:
```

Comment:
This block has been entirely and definitely removed because of this: https://github.com/travis-ci/travis-ci/issues/9361